doc/download.xml

```diff
@@ -73,6 +78,26 @@ true
 </Description>
 </ManSection>
 
+<Subsection Label="subsec-DownloadVerifyCertificate">
+<Heading>User preference <C>DownloadVerifyCertificate</C></Heading>
+<Index Key="DownloadVerifyCertificate"><C>DownloadVerifyCertificate</C></Index>
+
+The value <K>true</K> (the default) means that the server's certificate
+is checked in calls of <Ref Func="Download"/>,
+such that nothing gets downloaded if the certificate is invalid.
+
+If the value is <K>false</K> then those download methods that are based on
+<C>curl</C> or <C>wget</C> will omit the check of the server's certificate.
```

Comment:
I would prefer to avoid referring directly to specific download methods
whenever possible. E.g. like this:
```suggestion
If the value is <K>false</K> then download methods are supposed to omit
the check of the server's certificate (this may not be supported by all
download methods).
```